File: app/src/main/java/com/example/owner_pc/animechecker/contract/AnimeListFragmentContract.java
package com.example.owner_pc.animechecker.contract;

import com.example.owner_pc.animechecker.model.entity.AnimeCard;
import com.example.owner_pc.animechecker.model.entity.AnimePage;

import java.util.List;

/**
 * Created by owner-PC on 2017/05/29.
 */

public interface AnimeListFragmentContract {

//    void showAnimes(List<AnimeCard> animes);

    void showError();

    void itemClicked(AnimePage animePage);
}

File: app/src/main/java/com/example/owner_pc/animechecker/contract/AnimeListFragmentContract.kt
package com.example.owner_pc.animechecker.contract

import com.example.owner_pc.animechecker.model.entity.AnimePage

/**
 * Created by owner-PC on 2017/05/29.
 */

interface AnimeListFragmentContract {

    //    void showAnimes(List<AnimeCard> animes);

    fun showError()

    fun itemClicked(animePage: AnimePage)
}
